src/app/modules/dashboard/views/mission/mission-create/mission-create.component.ts

```diff
@@ -68,12 +68,16 @@ export class MissionCreateComponent implements OnInit {
   public form: FormGroup;
 
   constructor(
+    private NotificationService: NotificationService,
     private companies: CompanyService,
     private missions: MissionService,
     public store: StoreService,
     private router: Router,
     private fb: FormBuilder
-  ) {}
+  ) {
+    this.router.routeReuseStrategy.shouldReuseRoute = () => false;
```

Comment:
What's the reasoning behind this change?

Reply:
This is for when users click on "Create mission" while editing. Since both pages are on the same route, adding this line makes the component reload (clear form data) when going on the same route. Without this, the form data would not clear.